nix/nixos/cardano-tracer-service.nix

```diff
@@ -41,6 +41,11 @@ let serviceConfigToJSON =
           epHost    = "127.0.0.1";
           epPort    = 3200; ## supervisord.portShiftPrometheus
         } // (cfg.prometheus or {});
+        # Just an example for metrics compatibility mapping
+        # metricsComp = {
+        #     "Mempool.TxsInMempool" = "Mempool.TxsInMempool.Mapped";
+        #     "ChainDB.SlotNum" = "ChainDB.SlotNum.Mapped";
```

Comment:
@jutaro, just one thing -- can we change names in this example to make it more clear -- what metric already exists, and what will be created?   If I didn't have the context, I'd be a bit confused by it.

Also, another question -- can we flip the order?
It's a bit weird that the created metrics are on the right of what reads like an assignment operator.

Reply:
What names to you think make it clearer. I thought adding the ".Mapped" prefix makes it clear.

Reply:
It is a mapping from current name to new name. I think this is pretty straightforward, and reversing it would not make it clearer in my view ?!?

Reply:
It all hinges on how you read the assignment operator; it's meaning for ("X" = "Y") being 
> "X" has alias "Y"

right?
Without knowing how the resulting `Map Text Text` is applied in code, it's not straightforward; it would be nice to have that clarification in the comment, for someone just wanting to adjust the `.nix` service definition.

Reply:
There is a lot of mathematical baggage, where we define:
- something on the left
- to be equal to something on the right

..which, on the surface might feel like it's imperative PL bullshit, but it actuall runs deeper.

Can you think of a single definition where the "new thing" appears on the right?

Reply:
There are further reasons, which might be silly in this specific context -- but consider this:

- maps are asymmetric -- the thing on the left is unique, the thing on the right not so
- it makes sense to allow multiple aliases for the same source metric
- it doesn't make sense to have a single alias to be related to several source metrics

The status quo allows the strange thing and disallows the silly-but-meaningful thing.